schema.sql

```diff
@@ -9,4 +9,46 @@ CREATE TABLE animals (
   neutered BOOLEAN,
   weight_kg DECIMAL (5,2),
   PRIMARY KEY (id);
-);
```

Comment:
- [ ] Kindly remove the **_semi-column (;)_** at the end of the line of code `PRIMARY KEY (id)`. **_Semi-colons (;)_** are used to indicate the end of a statement in SQL. If you run this file, you will get a syntax error. Kindly remove it. 👍

Reply:
Ohh I didn't notice that. I removed it. 😊